test/support/fixture_helper.rb

```diff
@@ -0,0 +1,133 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require "rails_test_helper"
+
+module FixtureHelper
+  TEMP_FIXTURE_DIR = ROOT.join("tmp", "fixtures").to_s
+  FileUtils.mkdir_p(TEMP_FIXTURE_DIR)
+
+  Minitest.after_run do
+    # This does not get called on early exit, e.g. `Ctrl-C` or `(byebug) exit`.
+    FileUtils.remove_entry(TEMP_FIXTURE_DIR, true)
+  end
+
+  def around
+    old_working_dir = Dir.pwd
+    cache_rails_paths
+
+    yield
+
+    restore_rails_paths
+    Dir.chdir(old_working_dir)
```

Comment:
If we don't change the working directory, then parts of Packwerk use the `packwerk` project directory as the application root, not `tmp/fixtures/the_current_test`. Is it possible to always use Configuration.root_path as the application folder?